addons/scratch-notifier/addon.json

```diff
@@ -9,7 +9,9 @@
       "name": "NitroCipher/ZenithRogue"
     },
     {
-      "name": "notificationsounds.com (Scratch Addons ping)"
+      "name": "notificationsounds.com",
+      "note": "Scratch Addons ping",
+      "id": "Scratch Addons ping"
```

Comment:
does this work with non-alphanumeric characters?

Reply:
Yeah this was unintentional